cclib/method/volume.py

```diff
@@ -224,11 +253,11 @@ def electrondensity(coords, mocoeffslist, gbasis, volume):
                     for j, yval in enumerate(y):
                         tmp = []
                         for zval in z:
-                            tmp.append(bfs[bs].amp(xval, yval, zval))
-                        data[i, j, :] = tmp
-                data *= mocoeff[bs]
+                            tmp.append(bfs[bs].amp(xval, yval, zval)) # first get amplitudes
+                        data[i, j, :] = tmp # this contains the amplitudes
+                data *= mocoeff[bs] # then multiply with mocoeffs until homos

```

Comment:
I've noticed a bunch of these comments, but don't think they add much. Did you have a hard time understanding the code? I'd rather have a descriptive code comment toward the top of the function in that case.

Reply:
No -- I've written those when I was reading it and forgot to remove them back. Thanks.